Crash report 3.2: NullPointerException in NotificationsListFragment.onResume

```
java.lang.RuntimeException: Unable to resume activity {org.wordpress.android/org.wordpress.android.ui.notifications.NotificationsActivity}: java.lang.NullPointerException
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2575)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2603)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2089)
       at android.app.ActivityThread.access$600(ActivityThread.java:130)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4745)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: java.lang.NullPointerException
       at org.wordpress.android.ui.notifications.NotificationsListFragment.onResume(NotificationsListFragment.java:85)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:874)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1035)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1017)
       at android.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:1812)
       at android.app.Activity.performResume(Activity.java:5092)
       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2565)
       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2603)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2089)
       at android.app.ActivityThread.access$600(ActivityThread.java:130)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4745)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java
@@ -105,7 +105,7 @@ public class NotificationsListFragment extends ListFragment implements Bucket.Li
         }
 
         mFauxPullToRefreshHelper.unregisterReceiver(getActivity());
-        super.onDestroyView();
+        super.onDestroy();
     }
 
     private void initPullToRefreshHelper() {

